README.md

```diff
@@ -71,6 +71,9 @@ This is just a simple delegating method that explicitly calls
 Futures.transform(future, Function). There is also a corresponding
```

Comment:
No that is correct. It is `Futures.transform(future, Function)` but `Futures.transformAsync(future, AsyncFunction)`. They used to have `Futures.transform` overloaded with both Function and AsyncFunction arguments. In Guava 19 they added transformAsync and deprecated the transform variant that took AsyncFunction in preparation for removing it in Guava 20.